lib/Minz/FrontController.php

```diff
@@ -31,9 +31,6 @@ class Minz_FrontController {
 	 */
 	public function __construct () {
 		try {
-			Minz_Configuration::register('system',
-			                             DATA_PATH . '/config.php',
-			                             FRESHRSS_PATH . '/config.default.php');
```

Comment:
@marienfressinaud It was not so nice to initialise the system configuration here with a FreshRSS constant in Minz. What do you think?

Reply:
I agree, an alternative would be to rename the constant to something like `ROOT_PATH` (or create one)